samples/nrf9160/http_application_update/src/main.c

```diff
@@ -101,9 +101,9 @@ static int dfu_button_init(void)
 }
 
 
-void fota_dl_handler(enum fota_download_evt_id evt_id)
+void fota_dl_handler(const struct fota_download_evt *evt)
 {
-	switch (evt_id) {
+	switch (evt->id) {
```

Comment:
Please split this into a separate commit.

Reply:
done